.github/workflows/josh-json.yml

```
@@ -0,0 +1,35 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Josh - JSON - Node.js CI
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [10.x, 12.x, 14.x]
```

```suggestion
        node-version: 14.x
```
Is there a reason why we are testing Nodejs `v10.x` and `v12.x`, naturally LTS should be supported. Not to mention there are already places we are using `v14.x` syntax several times in the repository.

---

If there were places then it would have failed but sure